tabular/src/autogluon/tabular/configs/config_helper.py

```diff
@@ -0,0 +1,357 @@
+        If a key in `ag_args_fit` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+        """
+        self.config['ag_args_fit'] = ag_args_fit
+        return self
+
+    def ag_args_ensemble(self, ag_args_ensemble: dict) -> ConfigBuilder:
+        """
+        Keyword arguments to pass to all models.
+        See the `ag_args_ensemble` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+        Identical to specifying `ag_args_ensemble` parameter for all models in `hyperparameters`.
+        If a key in `ag_args_ensemble` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+        """
+        self.config['ag_args_ensemble'] = ag_args_ensemble
+        return self
+
+    def excluded_model_types(self, models: Union[str, list]) -> ConfigBuilder:
+        """
+        Banned subset of model types to avoid training during `fit()`, even if present in `hyperparameters`.
+        Reference `hyperparameters` documentation for what models correspond to each value.
+        Useful when a particular model type such as 'KNN' or 'custom' is not desired but altering the `hyperparameters` dictionary is difficult or time-consuming.
+            Example: To exclude both 'KNN' and 'custom' models, specify `excluded_model_types=['KNN', 'custom']`.
+        """
+        valid_keys = [m for m in MODEL_TYPES.keys() if m not in ['ENS_WEIGHTED', 'SIMPLE_ENS_WEIGHTED']]
+        if not isinstance(models, list):
+            models = [models]
+        for model in models:
+            assert model in valid_keys, f'{model} is not one of the valid models {valid_keys}'
+        self.config['excluded_model_types'] = sorted(list(set(models)))
+        return self
+
+    def included_model_types(self, models: Union[str, list]) -> ConfigBuilder:
+        """
+        Subset of model types to train during `fit()`.
+        Reference `hyperparameters` documentation for what models correspond to each value.
+        Useful when only the particular models should be trained such as 'KNN' or 'custom', but altering the `hyperparameters` dictionary is difficult or time-consuming.
+            Example: To keep only 'KNN' and 'custom' models, specify `included_model_types=['KNN', 'custom']`.
+        """
+        valid_keys = [m for m in MODEL_TYPES.keys() if m not in ['ENS_WEIGHTED', 'SIMPLE_ENS_WEIGHTED']]
+        if not isinstance(models, list):
+            models = [models]
+
+        unknown_keys = [k for k in models if k not in valid_keys]
+        assert len(unknown_keys) == 0, f'The following model types are not recognized: {unknown_keys} - use one of the valid models: {valid_keys}'
+
+        models = [m for m in valid_keys if m not in models]
+        self.config['excluded_model_types'] = models
+        return self
```

Comment:
Why set excluded_model_types? Does this mean someone cannot specify included_model_types in `.fit` directly?

Reply:
Correct: we don't have `included_model_types` in `.fit`. This is convenience method built on top of existing API capabilities.

Reply:
why force users to use the builder API to get access to `included_model_types`?

Reply:
IMHO neither methods should be actually a part of `.fit` API and should be handled via helpers, otherwise you will be forced changing .fit API every time a new kind of filter is wanted.

Reply:
Good points. I think it is fine as is then.